### DISCUSSION TITLE
How to search for only one product, like rtx 3080 only?

### DISCUSSION THREAD

**Discussion by THarutyunyan** [jef/streetmerchant#1700] [Help] [answered]
I can't figure out what to do to make Street Merchant search for only 3080. Please help.

**Comment by wofnull** [comment, marked as answer]
rename the "dotenv-example" file to "dotenv" and edit it to your likings, 

SHOW_ONLY_BRANDS= ->is for specific brands
SHOW_ONLY_MODELS= -> is for specific models from a brand
SHOW_ONLY_SERIES= -> is for the series you are searching for
STORES= -> is the store you are searching in

for example:
SHOW_ONLY_BRANDS=
SHOW_ONLY_MODELS=
SHOW_ONLY_SERIES=3080, 3090
STORES=amazon,bandh
would start the search for any 3080 / 3090 card in the stores amazon.com and bandh.com

for a full list  of Brands / models / series / Stores visit @jef s homepage, there is anything documented :
https://jef.codes/streetmerchant/reference/filter/

If you want the card type only put out results for cards in stock under a specific amount of money:
you can use the following lines in the config file to let the bot ignore everything abouve amount x money

MAX_PRICE_SERIES_3080=
MAX_PRICE_SERIES_3090=

**Reply by THarutyunyan** [reply]
Thanks a lot!
